src/main/java/com/openlattice/chronicle/configuration/TwilioConfiguration.kt

```diff
@@ -0,0 +1,32 @@
+package com.openlattice.chronicle.configuration
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kryptnostic.rhizome.configuration.Configuration
+import com.kryptnostic.rhizome.configuration.ConfigurationKey
+import com.kryptnostic.rhizome.configuration.SimpleConfigurationKey
+import com.kryptnostic.rhizome.configuration.annotation.ReloadableConfiguration
+
+private const val CONFIG_FILE_NAME = "twilio.yaml"
+@ReloadableConfiguration(uri = CONFIG_FILE_NAME)
+
+class TwilioConfiguration @JsonCreator constructor(
+    @JsonProperty val sid: String,
+    @JsonProperty val token: String,
+    @JsonProperty val fromPhone: String
+) : Configuration {
+
+    @JsonIgnore
+    override fun getKey(): ConfigurationKey {
+        return Companion.key
```

Comment:
You should be able to just do `return key` (maybe companion block needs to be above this).